blocks/collections-content-source-block/sources/content-api-collections.js

```diff
@@ -1,26 +1,64 @@
+import axios from 'axios';
+import { CONTENT_BASE, ARC_ACCESS_TOKEN } from 'fusion:environment';
 import getResizedImageData from '@wpmedia/resizer-image-block';
 
 const params = {
   _id: 'text',
   content_alias: 'text',
   from: 'text',
   size: 'text',
+  getNext: 'text',
 };
 
-const resolve = (key = {}) => {
+const fetch = (key = {}) => {
   const {
     'arc-site': site,
     _id,
     content_alias: contentAlias,
     from,
     size,
+    getNext = 'false',
   } = key;
-  return `content/v4/collections?${_id ? `_id=${_id}` : `content_alias=${contentAlias}`}${site ? `&website=${site}` : ''}${from ? `&from=${from}` : ''}${size ? `&size=${size}` : ''}&published=true`;
+  let updatedSize = size;
+  // Max collection size is 20
+  // See: https://redirector.arcpublishing.com/alc/docs/swagger/?url=./arc-products/content-api.json
+  // Want to ensure size is capped at 20 to prevent an error.
+  if (updatedSize && updatedSize > 9) updatedSize = size < 20 ? size : 20;
+
+  return axios({
+    url: `${CONTENT_BASE}/content/v4/collections?${_id ? `_id=${_id}` : `content_alias=${contentAlias}`}${site ? `&website=${site}` : ''}${from ? `&from=${from}` : ''}${size ? `&size=${updatedSize}` : ''}&published=true`,
+    headers: {
+      'content-type': 'application/json',
+      Authorization: `Bearer ${ARC_ACCESS_TOKEN}`,
+    },
+    method: 'GET',
+  }).then(({ data: content }) => {
+    if (getNext === 'false') {
+      return content;
+    }
+
+    const existingData = content;
+
+    return axios({
+      url: `${CONTENT_BASE}content/v4/collections?${_id ? `_id=${_id}` : `content_alias=${contentAlias}`}${site ? `&website=${site}` : ''}&from=${updatedSize}${size ? `&size=${updatedSize}` : ''}&published=true`,
```

Comment:
@matthewroach : We also need a trailing slash here.

Reply:
Added